client/src/utils/special-wikis-list.js

```diff
@@ -37,7 +37,7 @@ export default [
 	},
 	{
 		id: 'mediawikiwiki',
-		domain: 'mediawiki.org',
+		domain: 'www.mediawiki.org',
```

Comment:
So what happens here is that if you type mediawiki on the `Wiki` field you don't actually get mediawiki. I think this is a problem 'cause not everyone will know to find it through www.

Reply:
well we would have to have some display domain or something because not having the `www` causes a redirect and a CORS issue. :(

Reply:
well, technically it doens't work either way as it is even if users can find it. Lets get this through and create a different ticket to deal with this edge case on the search.